src/AdminAcceptanceTests.Steps/Utils/UITest.cs

```diff
@@ -1,27 +1,29 @@
 ﻿using AdminAcceptanceTests.Actions;
 using AdminAcceptanceTests.Actions.Collections;
+using AdminAcceptanceTests.TestData;
 using OpenQA.Selenium;
-using System;
-using System.Collections.Generic;
 
 namespace AdminAcceptanceTests.Steps.Utils
 {
     public sealed class UITest
     {
-        internal string ConnectionString;
-        internal IWebDriver Driver;
-        internal PageActionCollection Pages;
-        internal readonly string Url;
+        internal string ConnectionString { get; set; }
+        internal IWebDriver Driver { get; set; }
+        internal PageActionCollection Pages { get; set; }
+        internal readonly string Url { get; set; }        
+        internal readonly User AdminUser { get; set; }
```

Comment:
Does this compile (I didn't think `readonly` could be applied to a property)? If not, obviously just needs the `set` removing to make read-only.